modules/gob-web/modules/course/revisions.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import type {Course as CourseType} from '@gob/types'
+
+type Props = {
+	course: CourseType,
+}
+
+const RevisionTitle = styled.div`
+	padding-top: 16px;
+	font-weight: bold;
+`
+
+const Summary = styled.summary`
+	outline-style: none;
+`
+
+export class Revisions extends React.Component<Props> {
+	render() {
+		const {course} = this.props
+
+		if (!course.revisions) {
+			return null
+		}
+
+		const revisions = [...course.revisions].reverse()
+
+		const RevisionsList = revisions.map((items, index) => {
+			let revisionDate = new Intl.DateTimeFormat('en-US').format(
+				new Date(revisions[index]['_updated']),
+			)
+
+			return (
+				<ul key={index}>
+					<RevisionTitle>{revisionDate}</RevisionTitle>
+					{Object.keys(items).map(
```

Comment:
Use lodash.toPairs here, instead of iterating over the keys, so that you have easy access to both the keys and values.

Reply:
`lodash.toPairs` throws `Object(...) is not a function`

Reply:
You will need to look at the documentation